pulsar-io/debezium/core/src/main/java/org/apache/pulsar/io/debezium/PulsarDatabaseHistory.java

```diff
@@ -52,12 +52,12 @@
 import org.apache.pulsar.client.api.Schema;
 
 /**
- * A {@link DatabaseHistory} implementation that records schema changes as normal pulsar messages on the specified
+ * A {@link SchemaHistory} implementation that records schema changes as normal pulsar messages on the specified
  * topic, and that recovers the history by establishing a Kafka Consumer re-processing all messages on that topic.
  */
 @Slf4j
 @ThreadSafe
-public final class PulsarDatabaseHistory extends AbstractDatabaseHistory {
+public final class PulsarDatabaseHistory extends AbstractSchemaHistory {
```

Comment:
Why is this change needed? I don't know Debezium very well, but it seems that "database history" and "schema history" are separate concepts.

Reply:
nvm. found the answer, [DBZ-5043 Replace 'database history' with 'database schema history'](https://github.com/debezium/debezium/commit/b6c31f3da5559777a42eada8ccc18cde7e8bbc1f)

Reply:
It might be more consistent naming if `PulsarDatabaseHistory` is renamed to `PulsarSchemaHistory`. For example, in Debezium, `KafkaDatabaseHistory` was renamed to `KafkaSchemaHistory`.